src/libraries/sfx.proj

```diff
@@ -21,6 +21,22 @@
 
   <!-- Generate the targeting pack's framework list so that out-of-band projects can leverage it. -->
   <Import Project="frameworklist.targets" />
+
+  <!-- Copy the intellisense file to the targeting pack directory. -->
+  <Target Name="CopyDocumentationFiles"
+          AfterTargets="Build"
+          Condition="'$(RefOnly)' != 'true'">
+    <ItemGroup>
+      <DocumentationFileToCopy Include="@(SharedFrameworkAssembly->Metadata('DocFileItem'))"
+                               Condition="!$(NetCoreAppLibraryNoReference.Contains('%(Filename);'))" />
```

Comment:
What does the `*NoReference` suffix mean?

Reply:
https://github.com/dotnet/runtime/blob/07def40a1d8a900aa6e88c548742e547a24c0e8f/src/libraries/NetCoreAppLibrary.props#L177-L183

That's all the assemblies that are part of the the shared framework but not exposed in the targeting pack. Mostly "Private" assemblies these days.

Reply:
"Private" assemblies are runtime assemblies only and hence don't participate in the design and compile time experience (which includes Intellisense).